README.md

```diff
@@ -22,3 +22,10 @@ Code contributors
 
 Check out [src/README](src/README.md)
 
+### Drivers
+
+The drivers are located in this repository, for more information see one of the following:
+
+- [JavaScript](drivers/javascript/README.md)
```

Comment:
Consider linking to `drivers/javascript` instead of `drivers/javascript/README.md`. If GitHub finds a `README.md` file in a given directory, it will automatically display that file below the directory listing; for example, https://github.com/rethinkdb/rethinkdb/tree/next/drivers/javascript will show the JavaScript `README.md` file. So if this link points at the directory, users will see both the driver source code and the driver readme in the same page, which I think is slightly more convenient than just showing the driver readme.